src/fvectorbin/nnet3-fvector-perturb-signal.cc

```diff
@@ -44,27 +116,27 @@ int main(int argc, char *argv[]) {
     using namespace kaldi;
 
     const char *usage =
-        "Perturb the wave files supplied via the specified noise-range file\n"
+        "Perturb the wave files supplied via the specified noise-range\n"
         "Usage:  nnet3-fvector-perturb-signal [options...] <wav-in-rxfilename> "
         "<wav-out-wxfilename>\n"
         "e.g.\n"
-        "nnet3-fvector-perturb-signal --noise-range-file=uttid.range.n --add-noise-list="
-        "scp:noise.scp --input-channel=0 input.wav output.wav\n";
+        "nnet3-fvector-perturb-signal --noise=scp:noise.scp --noise-range="
+        "\"head -n 5 a.noiserange | tail -n 1\" --input-channel=0 input.wav "
```

Comment:
don't write head -n ... in the usage. Write actual values in range line for description.